Automattic/wp-calypso
# Add Promoted Posts Card to Marketing Tools

#### Proposed Changes

- Add a card to the top of the Tools->Marketing page in Calypso. Existing cards should be bumped down so this card is the first card to be shown. 
- Card title:  Get more traffic by promoting your content
- Card details text: Promote your content on millions of sites across WordPress.com and Tumblr with our new promoted post tool. Get new users to your best content, spending less than you would on a cup of coffee..
- Card CTA text: Go for it
- Card link: The button should link to the Advertising page /advertising/{site-slug}
- Request link pau2Xa-4qs-p2 

#### Before
<img width="1065" alt="Before" src="https://user-images.githubusercontent.com/11347472/194029331-98d92d6f-1a23-4a88-a98f-bda228f40d91.png">

#### After
<img width="1058" alt="r2-new-card-desktop" src="https://user-images.githubusercontent.com/11347472/194891957-feefeac8-ad02-4dc4-b4c6-ef057c801a14.png">

<img width="970" alt="r2-new-card-ipad" src="https://user-images.githubusercontent.com/11347472/194891993-e70fbc3a-3517-4994-b43c-b5e3acc384a1.png">


<img width="843" alt="r2-new-card-iphone" src="https://user-images.githubusercontent.com/11347472/194892152-57f31b90-416d-4a72-85d6-dae19cf898d2.png">


#### Testing Instructions
- On a calypso dashboard, go to Tools->Marketing /marketing/tools/{site-slug} in the sidebar menu.
- You should see the "Get more traffic by promoting your content" card as the first card right after the "Explore our premium plugin" header.
- Click the card's CTA button with the text "Go for it"
- Clicking the link should open the Advertising page /advertising/{site-slug} in a new tab.


#### Pre-merge Checklist






Related to #
